tests/helpers/datamodules.py

```diff
@@ -102,6 +101,8 @@ def sample(self):
 
 class ClassifDataModule(SklearnDataModule):
     def __init__(self, num_features=32, length=800, num_classes=3, batch_size=10):
+        if not _SKLEARN_AVAILABLE:
+            pytest.skip("`sklearn` is not available.")
```

Comment:
Why is the following condition required here if it's subclassing `SklearnDataModule`?

Reply:
Because it calls `super().__init__` only after the `make_classification()` call which comes from sklearn